EIPS/eip-1679.md

```diff
@@ -24,50 +24,49 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Ista
 
 ### Accepted EIPs
 
-- TBD
+- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs
+- [EIP-1702](https://eips.ethereum.org/EIPS/eip-1702): Generalized account versioning scheme
+- [EIP-2024](https://ethereum-magicians.org/t/blake2b-f-precompile/3157): Blake2b Precompile
+- [EIP-2028](https://eips.ethereum.org/EIPS/eip-2028): Calldata gas cost reduction
 
 ### Rejected EIPs
 
-- TBD
+- [EIP-615](https://eips.ethereum.org/EIPS/eip-615): Subroutines and Static Jumps for the EVM - *Withdrawn*
+- [EIP-663](https://eips.ethereum.org/EIPS/eip-663): Unlimited SWAP and DUP instructions
+- [EIP-1109](https://eips.ethereum.org/EIPS/eip-1109): PRECOMPILEDCALL opcode (Remove CALL costs for precompiled contracts)
+  - requirement of EIP-1962
+- [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559): Fee market change for ETH 1.0 chain
+- [EIP-1959](https://eips.ethereum.org/EIPS/eip-1959): New Opcode to check if a chainID is part of the history of chainIDs - *Withdrawn*
+- [EIP-1965](https://eips.ethereum.org/EIPS/eip-1965): Method to check if a chainID is valid at a specific block Number
+- [EIP-1985](https://eips.ethereum.org/EIPS/eip-1985): Sane limits for certain EVM parameters
+- [EIP-2014](https://eips.ethereum.org/EIPS/eip-2014): Extended State Oracle
+- [EIP-2025](https://eips.ethereum.org/EIPS/eip-2025): Funding Eth1.x with Developer Block Rewards *Withdrawn*
+- [EIP-2026](https://eips.ethereum.org/EIPS/eip-2026): State Rent H - Fixed Prepayment for accounts
+- [EIP-2029](https://eips.ethereum.org/EIPS/eip-2029): State Rent A - State counters contract
+  - requirement of EIP-2031
+- [EIP-2031](https://eips.ethereum.org/EIPS/eip-2031): State Rent B - Net transaction counter
+- [EIP-2035](https://eips.ethereum.org/EIPS/eip-2035): Stateless Clients - Repricing SLOAD and SSTORE to pay for block proofs
 
 ### Proposed EIPs
 
-- [EIP-615](https://eips.ethereum.org/EIPS/eip-615): Subroutines and Static Jumps for the EVM
-- [EIP-663](https://eips.ethereum.org/EIPS/eip-663): Unlimited SWAP and DUP instructions
 - [EIP-1057](https://eips.ethereum.org/EIPS/eip-1057): ProgPoW, a Programmatic
   Proof-of-Work
   - There is a
     [pending audit](https://medium.com/ethereum-cat-herders/progpow-audit-goals-expectations-75bb902a1f01),
     above and beyond standard security considerations, that should be evaluated
     prior to inclusion.
-- [EIP-1108](https://eips.ethereum.org/EIPS/eip-1108): Reduce alt_bn128 precompile gas costs
-- [EIP-1109](https://eips.ethereum.org/EIPS/eip-1109): PRECOMPILEDCALL opcode (Remove CALL costs for precompiled contracts)
-  - requirement of EIP-1962
 - [EIP-1283](https://eips.ethereum.org/EIPS/eip-1283): Net gas metering for SSTORE without dirty maps
```

Comment:
Not sure if this was properly accepted, but I think it was tentatively together with 1706 (and 2200).

Reply:
I left the tentative yes's out. I can add them to a Tentative yes list, or hopefully they get accepted today.